common/watcom.mif

```diff
@@ -1,48 +1,68 @@
 # Common elements for the Watcom makefiles
 
-CFLAGS += -bt=$(TARGET) -wx -zq -i=$(PDCURSES_SRCDIR)
+!ifdef __UNIX__
+DEL	= rm -f
+COPY	= cp
+!else
+DEL	= del
+COPY	= copy
+!endif
 
-!ifeq DEBUG Y
-CFLAGS		+= -d2 -DPDCDEBUG
-LDFLAGS		= D W A op q sys $(LTARGET)
+!ifeq TARGET linux
+.EXTENSIONS :
+.EXTENSIONS : . .lib .obj .h .c
 !else
-CFLAGS		+= -oneatx
-LDFLAGS		= op q sys $(LTARGET)
+E = .exe
 !endif
 
-!ifeq TARGET dos
-CFLAGS += -i="$(%WATCOM)/h"
-!else ifeq TARGET os2
-CFLAGS += -i="$(%WATCOM)/h" -i="$(%WATCOM)/h/os2"
-!else ifeq TARGET nt
-CFLAGS += -i="$(%WATCOM)/h" -i="$(%WATCOM)/h/nt"
+!ifeq MODEL f
+CC = *wcc386
+!else
+CC = *wcc
 !endif
 
-RM		= del
+LIBEXE = *wlib -q -n -b -c -t
+
+LINK = *wlink
+
+CFLAGS += -bt=$(TARGET) -m$(MODEL) -wx -zq -i=$(PDCURSES_SRCDIR)
+!ifeq DEBUG Y
+CFLAGS += -d2 -DDEBUG -DPDCDEBUG
+!else
+CFLAGS += -s -oneatx -DNDEBUG
+!endif
+!ifdef CHTYPE_32
+CFLAGS += -DCHTYPE_32
+!endif
+!ifeq TARGET dos
```

Comment:
Shouldn't the WIDE definition moved here, too?
... that's the last issue/question I have with this PR ;-)

Reply:
I am sorry I still don't finished with PR
I think there are multiple macros which should be added
PDC_WIDE
PDC_FORCE_UTF8
PDC_DLL_BUILD
PDC_RGB

Reply:
No problem, just drop a note when you're finished, or even better do this with [re-]questing a review.

Reply:
I will do.